apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="date-range-demo">
+  <form id="screenshot-date-range-picker" novalidate [formGroup]="reactiveForm">
+    <div class="sky-form-group">
+      <sky-date-range-picker
+        formControlName="lastDonation"
+        label="Last donation"
+        [calculatorIds]="calculatorIds"
+        [dateFormat]="dateFormat"
+        [endDateRequired]="endDateRequired"
+        [startDateRequired]="startDateRequired"
+      >
+      </sky-date-range-picker>
+    </div>
+
+    <div class="sky-form-group">
+      <button class="sky-btn sky-btn-default" type="button" (click)="submit()">
+        Submit
+      </button>
+    </div>
+  </form>
+
+  <h3>&lt;sky-date-range-picker&gt; statuses</h3>
+
+  <table>
+    <tr>
+      <th>Touched</th>
+      <td>{{ pickerFormControl.touched }}</td>
+    </tr>
+    <tr>
+      <th>Pristine</th>
+      <td>{{ pickerFormControl.pristine }}</td>
+    </tr>
+    <tr>
+      <th>Valid</th>
+      <td>{{ pickerFormControl.valid }}</td>
+    </tr>
+  </table>
+
+  <div *ngIf="pickerFormControl.errors" class="sky-error-label">
+    {{ pickerFormControl.errors | json }}
+  </div>
+
+  <h3>&lt;sky-date-range-picker&gt; value</h3>
+
+  <!-- <pre><code style="display:block;">calculatorId: {{ reactiveForm.value.lastDonation?.calculatorId }}
```

Comment:
This comment block still needed?

Reply:
I fixed this. The date pipe can't take in invalid strings and it was causing an issue. So I had commented it out. I changed this to only show the values when it is valid. Not too worried about it since it is just the playground.